include/maths/CBoostedTreeImpl.h

```diff
@@ -120,11 +123,13 @@ class MATHS_EXPORT CBoostedTreeImpl final {
     //! Visit this tree trainer implementation.
     void accept(CBoostedTree::CVisitor& visitor);
 
+    //! \return Class containing best hyperparameters.
```

Comment:
nit: strictly this is an object of the class but how about
```suggestion
    //! \return The best hyperparameters for validation error found so far.
```